edgelet/build/windows/openssl.ps1

```diff
@@ -2,39 +2,47 @@
 
 function Get-OpenSSL
 {
-    if (!((Test-Path -Path $env:HOMEDRIVE\vcpkg) -and ((Test-Path -Path $env:HOMEDRIVE\vcpkg\vcpkg.exe))))
+    param(
+        [switch]$Arm
+    )
+
+    $ErrorActionPreference = 'Continue'
+
+    if (!((Test-Path -Path $env:SystemDrive\vcpkg) -and ((Test-Path -Path $env:SystemDrive\vcpkg\vcpkg.exe))))
     {
         Write-Host "Installing vcpkg from github..."
-        git clone https://github.com/Microsoft/vcpkg $env:HOMEDRIVE\vcpkg
+        git clone https://github.com/Microsoft/vcpkg $env:SystemDrive\vcpkg
```

Comment:
Why do you need to change this? The script is intended for everyone to use, not just the VSTS agents, and users will probably not have access to write to `C:\vcpkg`

Reply:
When I tried on my VSTS server, $env:HOMEDRIVE was not defined, so I had to change to something that works. What's the recommended var to use here? I can change back to HOMEDRIVE and try it on MsAzure pipeline

Reply:
Reverted back to $env:HOMEDRIVE